src/components/TimePicker/__test__/ampmSelect.spec.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import AmPmSelect from './../ampmSelect';
+
+describe('<AmPmSelect/>', () => {
+    it('should render the right markup initially when component is not focused', () => {
+        const component = mount(<AmPmSelect tabIndex="-1" />);
+        component.setState({ isFocused: false });
```

Comment:
we should not set isFocused to false here since the test is for check what is render initially and therefore this implies that we are also testing that initially the isFocused is false, but if you manually set isFocused to false then this test no make sense